src/theory/quantifiers/quantifiers_rewriter.h

```diff
@@ -267,27 +273,18 @@ class QuantifiersRewriter : public TheoryRewriter
    *   (or (P x z) (not (Q y z)))
    * and add {x} to args, and {y} to nargs.
    */
-  static Node computePrenex(Node q,
-                            Node body,
-                            std::unordered_set<Node>& args,
-                            std::unordered_set<Node>& nargs,
-                            bool pol,
-                            bool prenexAgg);
-  static Node computeSplit( std::vector< Node >& args, Node body, QAttributes& qa );
-private:
- static Node computeOperation(Node f,
-                              RewriteStep computeOption,
-                              QAttributes& qa);
-
-public:
- RewriteResponse preRewrite(TNode in) override;
- RewriteResponse postRewrite(TNode in) override;
+  Node computePrenex(Node q,
+                     Node body,
+                     std::unordered_set<Node>& args,
+                     std::unordered_set<Node>& nargs,
+                     bool pol,
+                     bool prenexAgg) const;
+  Node computeSplit(std::vector<Node>& args, Node body, QAttributes& qa) const;
 
-private:
-  /** options */
- static bool doOperation(Node f, RewriteStep computeOption, QAttributes& qa);
+ public:
+  RewriteResponse preRewrite(TNode in) override;
+  RewriteResponse postRewrite(TNode in) override;
```

Comment:
Can you make sure that there is only one `public:` and one `private:` block?

Reply:
Yes, although it will be a big diff, I'll do this now.